docs/source/development/linting.md

```diff
@@ -1,33 +1,34 @@
-maintain the code quality. Linting makes your code more readable, easy to debug and maintain, and stylistically
-consistent.
 
-## Set up linting tools
-There are a variety of linting tools available to use with your Kedro projects. This guide shows you how to use
+Code formatting guidelines set a standard for the layout of your code, for stylistic elements such as use of line breaks and whitespace. Format doesn't have any impact on how the code works, but using a consistent style makes your code more readable, and makes it more likely to be reused.
+
+Linting tools check your code for errors such as a missing bracket or line indent. This can save time and frustration because you can catch errors in advance of running the code.
+
+As a project grows and goes through various stages of development it becomes important to maintain code quality. Using a consistent format and linting your code ensures that it is consistent, readable, and easy to debug and maintain.
+
+## Set up Python tools
+There are a variety of Python tools available to use with your Kedro projects. This guide shows you how to use
 [`black`](https://github.com/psf/black), [`flake8`](https://github.com/PyCQA/flake8), and
-[`isort`](https://github.com/PyCQA/isort) to lint your Kedro projects.
+[`isort`](https://github.com/PyCQA/isort) to format and lint your Kedro projects.
 - **`black`** is a [PEP 8](https://peps.python.org/pep-0008/) compliant opinionated Python code formatter. `black` can
 check for styling inconsistencies and reformat your files in place.
 [You can read more in the `black` documentation](https://black.readthedocs.io/en/stable/).
 - **`flake8`** is a wrapper around [`pep8`](https://pypi.org/project/pep8/),
-[`pyflakes`](https://pypi.org/project/pyflakes/), and [`mccabe`](https://pypi.org/project/mccabe/) which can flag
-programming errors and coding style inconsistencies with respect to [PEP 8](https://peps.python.org/pep-0008/),
+[`pyflakes`](https://pypi.org/project/pyflakes/), and [`mccabe`](https://pypi.org/project/mccabe/) which can lint code and format it with respect to [PEP 8](https://peps.python.org/pep-0008/),
 and check the cyclomatic complexity of your code base.
 [You can read more in the `flake8` documentation](https://flake8.pycqa.org/en/latest/).
-- **`isort`** is a Python library used to sort imports alphabetically and automatically separate them into sections by
+- **`isort`** is a Python library used to reformat code by sorting imports alphabetically and automatically separating them into sections by
 type. [You can read more in the `isort` documentation](https://pycqa.github.io/isort/).
 
-### Install linting tools
-You can install `black`, `flake8`, and `isort` by adding the following lines to your project's `src/requirements.txt`
+### Install the tools
+Install `black`, `flake8`, and `isort` by adding the following lines to your project's `src/requirements.txt`
```

Comment:
Now that I see this... it feels kind of wrong to add the linters to `src/requirements.txt`, shouldn't we have a `dev-requirements.txt` of sorts? cc @merelcht

Reply:
good push!

Reply:
We've talked about that before but I don't remember if we ever made a decision on it. Feel free to open an issue and we can make a proper decision this time 😄

Reply:
Done! https://github.com/kedro-org/kedro/issues/2519